test/Serialization/implementation-only-open.swift

```diff
@@ -0,0 +1,25 @@
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/swift)
+// RUN: %target-swift-frontend -emit-module -emit-module-path %t/swift/SwiftLibrary.swiftmodule %s -module-name SwiftLibrary -I%S/Inputs
+// RUN: %target-typecheck-verify-swift -DCLIENT -c %s -module-name client -I%t/swift
```

Comment:
What does `-c` do? I couldn't find it either options list.

Reply:
It's an alias for `-emit-object`: https://github.com/apple/swift/blob/ac875d00f04126d8a579e307d052df9752a18286/include/swift/Option/Options.td#L1015-L1016

Reply:
Thanks, should have looked at the modes too.